src/libs/blueprint/conduit_blueprint_mesh_utils.cpp

```diff
@@ -2014,37 +2014,35 @@ topology::unstructured::generate_offsets(const Node &n,
         Node &dest_elem_off = const_cast<Node &>(n)["elements/offsets"];
         Node &dest_subelem_off = const_cast<Node &>(n)["subelements/offsets"];
 
-        const Node& topo_elem_size = n["elements/sizes"];
-        const Node& topo_subelem_size = n["subelements/sizes"];
+        index_t_accessor topo_elem_size = n["elements/sizes"].as_index_t_accessor();
+        index_t_accessor topo_subelem_size = n["subelements/sizes"].as_index_t_accessor();
 
         Node elem_node;
         Node subelem_node;
 
-        std::vector<index_t> shape_array;
+        index_t elem_count = topo_elem_size.number_of_elements();
+        std::vector<index_t> shape_array(elem_count, 0);
         index_t ei = 0;
```

Comment:
We can avoid the std::vector all together (even with the std::vector there is still index_t type conversoin, so best to just use index_t):

```suggestion
dest_elem_off.set(DataType::index_t(elem_count);
index_t_array shape_array  = dest_elem_off.value();
```